<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-89HMT6O" failures="3" tests="49" name="TestRunner.runner" time="38.366" errors="0" timestamp="2023-07-27T14:25:55 GMT-04:00" skipped="0">
  <testcase classname="TestRunner.runner" name="runScenario" time="0.189"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.290"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.405"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.301"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="1.812"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.652"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.321">
    <failure message="expected [200] but found [400]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1418)
at org.testng.Assert.assertEquals(Assert.java:1448)
at stepDefinition.FAssignmentSubmission_SD.user_receives_created_status_with_response_body(FAssignmentSubmission_SD.java:140)
at ✽.User receives 200 Created Status with response body for assignmentSubmission(file:///C:/Users/Viru/Desktop/JavaPrograms/RestAssureApi/src/test/resources/featureFiles/05LMS_AssignmentSubmission.feature:47)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.333"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.764"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="2.819"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.226">
    <failure message="expected [404] but found [200]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [404] but found [200]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1418)
at org.testng.Assert.assertEquals(Assert.java:1448)
at stepDefinition.FAssignmentSubmission_SD.user_receives_created_status_with_response_body(FAssignmentSubmission_SD.java:140)
at ✽.User receives 404 Created Status with response body for assignmentSubmission(file:///C:/Users/Viru/Desktop/JavaPrograms/RestAssureApi/src/test/resources/featureFiles/05LMS_AssignmentSubmission.feature:88)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.410"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.477"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.209"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.380"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.330"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.485"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.259"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.441"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.526"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="11.748"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.506"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.728"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.807"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.238"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.422"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.189"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.525"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.587"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.233"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="2.281"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.352"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.439"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.240"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.216"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.207"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="1.393"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.255"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.415"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.202"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="1.749"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.250"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.384"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.237"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.572"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.274">
    <failure message="expected [400] but found [200]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [400] but found [200]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1418)
at org.testng.Assert.assertEquals(Assert.java:1448)
at stepDefinition.FAssignmentSubmission_SD.user_receives_created_status_with_response_body(FAssignmentSubmission_SD.java:140)
at ✽.User receives 400 Created Status with response body for assignmentSubmission(file:///C:/Users/Viru/Desktop/JavaPrograms/RestAssureApi/src/test/resources/featureFiles/05LMS_AssignmentSubmission.feature:163)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.397"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.474"/>
  <system-out/>
  <testcase classname="TestRunner.runner" name="runScenario" time="0.417"/>
  <system-out/>
</testsuite> <!-- TestRunner.runner -->
